src/open_company_web/components/finances.cljs

```diff
@@ -15,40 +24,112 @@
           costs (:costs finances)
           currency (:currency data)
           burn-rate (- revenue costs)
-          burn-rate-label (if (> burn-rate 0) "Growth rate: " "Burn rate: ")
-          burn-rate-classes (str "num " (if (> burn-rate 0) "green" "red"))
-          profitable (if (> burn-rate 0) "Yes" "No")
-          run-away (if (<= burn-rate 0) (quot cash burn-rate) "N/A")
-          currency-symbol (get-symbols-for-currency-code currency)]
-      (r/well {:class "report-list finances clearfix"}
-        (dom/div {:class "report-list-left"}
-          (om/build report-editable-line {
-            :cursor (:finances data)
-            :key :cash
-            :prefix currency-symbol
-            :label "cash on hand"
-            :pluralize false})
-          (om/build report-editable-line {
-            :cursor (:finances data)
-            :key :revenue
-            :prefix currency-symbol
-            :label "revenue this month"
-            :pluralize false})
-          (om/build report-editable-line {
-            :cursor (:finances data)
-            :key :costs
-            :prefix currency-symbol
-            :label "costs this month"
-            :pluralize false})
-          (dom/div
-            (dom/span {:class "label"} (str "Profitable this month? " profitable)))
-          (dom/div
-            (dom/span {:class "label"} burn-rate-label)
-            (dom/span {:class "label"} currency-symbol)
-            (dom/span {:class burn-rate-classes} (thousands-separator (abs burn-rate))))
-          (dom/div
-            (dom/span {:class "label"} "Runaway: " (if (<= burn-rate 0) (str (abs run-away) " months") "N/A")))
-          (om/build comment-component {:cursor finances :key :comment}))))))
+          burn-rate (if (js/isNaN burn-rate) 0 burn-rate)
+          positive-diff (>= burn-rate 0)
+          burn-rate-label (if positive-diff "Growth rate" "Burn rate")
+          burn-rate-classes (str "num " (if positive-diff "green" "red"))
+          burn-rate-helper (if positive-diff "Cash earned this quarter" "Cash used this quarter")
+          profitable (if positive-diff "Yes" "No")
+          run-away (if (not positive-diff) (quot cash burn-rate) "N/A")
+          currency-dict (utils/get-currency currency)
+          currency-symbol (utils/get-symbol-for-currency-code currency)]
+      (p/panel {:header (dom/h3 "Finances") :class "finances clearfix"}
+        (dom/div {:class "finances row"}
+          (dom/form {:class "form-horizontal"}
+
+            ;; Cash
```

Comment:
@bago2k4 Like with compensation, seems Cash/Revenue/Cost should be moved out to their own function at a minimum and DRY'd up ideally.